iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -535,6 +538,38 @@ public void testCreateTableWithoutColumnComments() {
     }
   }
 
+  @Test
+  public void testAlterTableProperties() {
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+    shell.executeStatement("CREATE EXTERNAL TABLE customers (" +
+        "t_int INT,  " +
+        "t_string STRING) " +
+        "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
+        testTables.locationForCreateTableSQL(identifier) +
+        testTables.propertiesForCreateTableSQL(ImmutableMap.of()));
+    String propKey = "dummy";
+    String propValue = "dummy_val";
+    shell.executeStatement(String.format("ALTER TABLE customers SET TBLPROPERTIES('%s'='%s')", propKey, propValue));
+    // Check the Iceberg table parameters
+    Table icebergTable = testTables.loadTable(identifier);
+    Assert.assertTrue(icebergTable.properties().containsKey(propKey));
+    Assert.assertEquals(icebergTable.properties().get(propKey), propValue);
+    propValue = "new_dummy_val";
+    shell.executeStatement(String.format("ALTER TABLE customers SET TBLPROPERTIES('%s'='%s')", propKey, propValue));
```

Comment:
nit: for readability, would it make sense to separate the 3 sections with a comment, e.g.
`// adding a new property`, 
`// changing the value of an existing property`, 
`// removing/unsetting an existing prop` ..., etc.

Reply:
Fixed.